__tests__/babel-wrapper.js

```diff
@@ -0,0 +1,3 @@
+module.exports = require('babel-jest').createTransformer({
+  rootMode: 'upward',
```

Comment:
I don't think you need that file, it's only needed when running jest in a mono-repo (using yarn workspaces)

Reply:
<img width="568" alt="Screen Shot 2021-03-15 at 12 11 52 PM" src="https://user-images.githubusercontent.com/818848/111208153-a594be80-8587-11eb-858d-fc2a74fb95e3.png">

i get an error if i remove it

Reply:
correct done